src/website/app/demos/Dialog/examples/Dialog/modeless.js

```diff
@@ -0,0 +1,74 @@
+/* @flow */
+import { Component } from 'react';
+import Button from '../../../../../../library/Button';
+import Dialog from '../../../../../../library/Dialog';
+
+export default {
+  id: 'modeless',
+  title: 'Modeless',
+  description: ` Dialog has modal behavior by default but it can also be
+modeless, which changes its behavior:
+
+- The position is set to 'fixed'
```

Comment:
Not really sure that we need to call all of this stuff out.

Reply:
Yeah, it is a bit verbose and maybe overkill; that all came from my original description. What if we just said something like "Use the `modeless` prop to render a Dialog without disabling the rest of the app. Note that this form of Dialog cannot be dismissed without a closing action."

Reply:
Not sure I agree. I think it's helpful to know exactly how the behavior changes, as it could affect the app.